en/docs/cookies/zero.md

```diff
@@ -95,32 +98,57 @@ Then start training through [launch Module](../parallelism/04_launch.md)
+ZeRO can be enabled through the interface [config.set_zero_redundancy_optimizer_mode](https://oneflow.readthedocs.io/en/v0.8.1/generated/oneflow.nn.graph.graph_config.GraphConfig.enable_zero.html#oneflow.nn.graph.graph_config.GraphConfig.enable_zero) .
+
+#### Enable Stage 1 of ZeRO
 
 ```python
-self.config.set_zero_redundancy_optimizer_mode("distributed_split")
+class CustomGraph(flow.nn.Graph):
+    def __init__(self):
+        super().__init__()
+        ...
+        # Enable stage 1 of ZeRO
+        self.config.enable_zero(True, stage=1)
+        ...
 ```
 
 !!! Note
     When using the model for continuous training and prediction: After the training is performed once, ZeRO will automatically change the SBP parameter of the model from Broadcast to Split; when performing prediction, Split will be used for automatic inference without configuring ZeRO.
 
-!!! Warning
-    This API is subject to change in the future.
+#### Enable Stage 2 of ZeRO
 
-**Stage 2**: On the basis of stage 1, adding `flow.boxing.nccl.enable_use_compute_stream(True)` can enable stage 2. i.e. add the following code in the nn.Graph model above:
+```python
+class CustomGraph(flow.nn.Graph):
+    def __init__(self):
+        super().__init__()
+        ...
+        # Enable stage 2 of ZeRO
+        self.config.enable_zero(True, stage=2)
+        ...
+```
+
+Generally speaking, the optimization of stage 2 has large optimization of memory and small speed impact, so it is recommended to use stage 2 optimization. It can be enabled in a simpler way:
 
 ```python
-self.config.set_zero_redundancy_optimizer_mode("distributed_split")
-flow.boxing.nccl.enable_use_compute_stream(True)
+class CustomGraph(flow.nn.Graph):
+    def __init__(self):
+        super().__init__()
+        ...
+        # Enable stage 2 of ZeRO
+        self.config.enable_zero()
+        ...
 ```
 
-**Stage 3**:On the basis of stage 2, adding `flow.boxing.nccl.disable_group_boxing_by_dst_parallel(True)` can enable stage 3. i.e. add the following code in the nn.Graph model above:
+#### Enable Stage 3 of ZeRO
 
 ```python
-self.config.set_zero_redundancy_optimizer_mode("distributed_split")
-flow.boxing.nccl.enable_use_compute_stream(True)
-flow.boxing.nccl.disable_group_boxing_by_dst_parallel(True)
+class CustomGraph(flow.nn.Graph):
+    def __init__(self):
+        super().__init__()
+        ...
+        # Enable stage 3 of ZeRO
+        self.config.enable_zero(True, stage=3)
+        ...
 ```
 
-!!! Note
-    Although enabling the third stage can minimize the memory consumption, it will increase the communication cost, and the second stage is generally used in practice.
\ No newline at end of file
+Although enabling the third stage can minimize the memory consumption, it will increase the communication cost which will lead to lower speed.
```

Comment:
```suggestion
Although enabling the third stage can minimize the memory consumption, it will increase the communication cost which will bring lower speed.
```